_data/mentors.yml

```diff
@@ -1201,7 +1201,7 @@
 - name: Jyoti Yadav
   disabled: false
   matched: false
-  sort: 300
+  sort: 100
```

Comment:
I think we should add them as available. They just join as mentor, right? Did we asked mentors for availability?

Reply:
Same with Jyoti. I mainly just checked their `availability` field to update the sorts